chalice/build_deploy_config.sh

```diff
@@ -2,12 +2,12 @@
 
 set -euo pipefail
 
-if [[ $# != 1 ]]; then
-    echo "Usage: $(basename $0) stage"
+if [[ -z $DSS_DEPLOYMENT_STAGE ]]; then
+    echo 'Please run "source environment" in the data-store repo root directory before running this command'
```

Comment:
Note: during `make deploy` the argument will be printed to STDOUT, e.g. `./build_deploy_config.sh stage`. IMO this is preferable to jus seeing `./build_deploy_config.sh` float by.

Reply:
What do you suggest I do?